### DISCUSSION TITLE
Local and ICD Codes

### DISCUSSION THREAD

**Discussion by santaviccap** [CMSgov/price-transparency-guide#560] [Q&A] [answered]
Probably a simple 2 part question:

- What are local codes?  Quite familiar with hcpcs, CPT, DRG, and the other usual suspects.  Not following the 'local' code type. 
- Why are plans using ICD-10 codes?  Is this only for procedures not falling under a DRG?

**Comment by shaselton-usds** [comment, marked as answer]
Local codes can be internal coding systems to the organization. 

If you are seeing ICD codes, then whatever negotiated item/service is based on that.

**Comment by TiaGossSawhney** [comment]
To provide more context to the answer:  

1) Plans may cover services that are not described via current HCPCS codes (inclusive of Level 1 codes which are more commonly called CPT codes).  These services still have to be billed, so the plan creates it's own code.  Local codes are most common with Medicaid and Medicare managed care plans and Medicaid fee-for-service plans as they frequently cover non-medical services and HCPCS codes are designed to primarily described medical services.  

2)  "ICD" can refer to ICD-10-CM diagnosis codes or ICD-10-PCS procedure codes.   ICD-10-CM diagnosis codes are pervasive -- every health insurance claim has one or more ICD-10-CM diagnosis code.  ICD-10-PCS procedure codes are generally reported only within inpatient hospital claims where they are integral to assigning the MS-DRG.  

Outpatient services are reported using HCPCS codes. This means that the same procedure has a different code when it is performed inpatient vs. outpatient (where outpatient may be a hospital outpatient department, an ambulatory surgery center, or a physician office).  It's a real headache if you are trying to examine procedures that can be performed inpatient or outpatient as there is no clean mapping between ICD-10-CMS and  HCPCS.  There has been talk of using ICD-10-CM codes for outpatient surgeries.  

It is possible that a payer is using ICD-10-PCS as the basis for inpatient payment rather than MS-DRG.  It is possible that some payer(s), some where is using ID-10-CM codes as the basis for outpatient reimbursements.
